scripts/march-to-cpu-opt

```diff
@@ -146,6 +147,14 @@ def get_vlen(ext_dict):
             vlen = max(vlen, zvelen)
     return vlen
 
+def get_elen(ext_dict):
+    elen = 32
```

Comment:
You are right that the `zveN` extensions were ignored.
But why is `32` a better default than `xlen` (if no `zveN` extensions are defined)?

Maybe something like this would be reasonable:
```
elen = xlen
if "zve32x" in ext_dict or "zve32f" in ext_dict or "zve32d" in ext_dict:
    elen = 32
if "zve64x" in ext_dict or "zve64f" in ext_dict or "zve64d" in ext_dict:
    elen = 64
return elen
```

Reply:
Oh, I see. Let me update this soon.

Reply:
Tested and updated.